### PR TITLE
Feature/change signing to underwritter plus mandates

### PR DESCRIPTION
**Pull request by meletios**
- Redirects signing to underwriter instead of product-pricing
- ~~handles the creation of the member mandate~~

### CODE REVIEW

File: member-service/src/main/java/com/hedvig/memberservice/events/MemberSignedEvent.java [hunk 1]
```diff
@@ -18,4 +15,87 @@
...
+  @java.beans.ConstructorProperties({"id", "referenceId", "signature", "oscpResponse", "ssn"})
+  public MemberSignedEvent(Long id, String referenceId, String signature, String oscpResponse, String ssn) {
+    this.id = id;
+    this.referenceId = referenceId;
+    this.signature = signature;
+    this.oscpResponse = oscpResponse;
+    this.ssn = ssn;
+  }
+
+  public Long getId() {
```
Comment (AlexSJ1990) [inline member-service/src/main/java/com/hedvig/memberservice/events/MemberSignedEvent.java]: Why do we get rid of @ Value?

File: member-service/src/main/java/com/hedvig/memberservice/sagas/MemberSignedSaga.kt [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+    @SagaEventHandler(associationProperty = "id")
+    @StartSaga
+    @EndSaga
+    fun onMemberSignedEvent(e: MemberSignedEvent, eventMessage: EventMessage<MemberSignedEvent>) {
+
+        try {
+            underwriterApi.memberSigned(e.getId().toString(), e.getReferenceId(), e.getSignature(), e.getOscpResponse())
+        } catch (ex: RuntimeException) {
+            log.error("Could not notify underwriter about signed member for memberId: {}", e.getId(), ex)
```
Comment (AlexSJ1990) [inline member-service/src/main/java/com/hedvig/memberservice/sagas/MemberSignedSaga.kt]: shall we do log.error("Could not notify underwriter about signed member for memberId: %, exception is %", e.getId(), ex)

Comment (meletios) [reply member-service/src/main/java/com/hedvig/memberservice/sagas/MemberSignedSaga.kt]: Correct just fixed it

File: member-service/src/main/java/com/hedvig/memberservice/events/MemberSignedEvent.java (hunk 1 above)
Comment (meletios) [reply member-service/src/main/java/com/hedvig/memberservice/events/MemberSignedEvent.java]: Delombok it since we are using this event in Kotlin

**Pull request merged**